src/gltf-component-mappings.js

```diff
@@ -72,3 +72,22 @@ AFRAME.GLTFModelPlus.registerComponent("nav-mesh", "nav-mesh", (el, _componentNa
   // nav-mesh-helper will query for it later.
   el.setAttribute("nav-mesh");
 });
+
+AFRAME.GLTFModelPlus.registerComponent("pinnable", "pinnable");
+
+AFRAME.GLTFModelPlus.registerComponent("media", "media", (el, componentName, componentData) => {
```

Comment:
If we are going to make this sort of combo component I would propose we at least point it out a bit more by calling it something like `networked-media`

Reply:
as it's written right now, it's parameterized so that it could be networked or non-networked -- the intent was this was a generic catch-all component for any URLs you want to put into the scene (and perhaps the contract includes 'is resolved at load time vs scene export time')